### ISSUE TITLE
Feature Request - Media Issues

### ISSUE THREAD

**Issue by bbakermmc** [darkalfx/requestrr#14]
Ability to log issues with media and have it log to UI/Channel.

EX: !issue movie Bad Boys for Life
(Show Only Movies/TV that are available)  (If not found them maybe the issue types are "Obscure/NotFound" media request to be sourced not thru the arrs)
Select Movie
(Show list of configurable options)
     Audio Sync
     Not Right Movie 
     Quality
Select Reason
Add optional note?

Another command of !issues list 
(Show grid of issues)
Select Issue
Ability to resolve/delete from list

**Comment by darkalfx** [comment]
Ooh that’s interesting, I definitely didn’t think about that.

I quite like it, we’re eventually going to have a dashboard in the web portal a ton of information about what’s going on, it could be a good place to also display the issues.

**Comment by bbakermmc** [comment]
Yeah its a fairly common issue.  Does the UI have multi roles?  I just started looking at this project, and comparing it to "Mellow".  If the UI has multi roles it would be nice since you dont want all users changing config options, but maybe you want some people to help "source" the issues/foreign requests.

**Comment by darkalfx** [comment]
The UI is not currently multi roles, but it’s definitely an interesting feature, I’ll add it to the backlog.

The first version of this feature will probably be for admins only, but then we’ll see about potentially adding multiple roles later down the line.

**Comment by bbakermmc** [comment]
Thanks.  You might want to break out issues and not found requests, it "could" be two different audiences.

**Comment by nrgbistro** [comment]
It would also be cool to have the bot send a DM to the server maintainer/moderators about submitted issues (if that wasn't already assumed with the suggested implementation).

**Comment by julian-str** [comment]
Would this be compatible with Ombi Issues since the bot is already able to send requests to Ombi?

**Comment by M3th0s** [comment]
+1 for this!
